[Feature] Center slides in Carousel when less slides than slidesPerPage (Issue/3041) (#3372)
porsche-design-system/porsche-design-system: packages/components/CHANGELOG.md

- Styles: `theme{Light|Dark}BackgroundFrosted` and `$pds-theme-{light|dark}-background-frosted` color
  ([#3409](https://github.com/porsche-design-system/porsche-design-system/pull/3409))

> - `Carousel`: support for centering slides if `alignHeader` prop is set to `center` and amount of slides is less than

Reviewer: ```suggestion
- `Carousel`: slides are centered if `alignHeader` prop is set to `center` and amount of slides is less than
```
Should this be in the Changed block?
